integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/rest/test/header/ContentTypeTestCase.java

```diff
@@ -44,66 +47,72 @@
 import org.wso2.esb.integration.common.utils.ESBTestCaseUtils;
 
 public class ContentTypeTestCase extends ESBIntegrationTest {
+
     private Log log = LogFactory.getLog(ContentTypeTestCase.class);
-	private HttpServer server = null;
-	public static String contentType;
-
-	@BeforeClass(alwaysRun = true)
-	public void init() throws Exception {
-		super.init();
-		String relativePath = "/artifacts/ESB/synapseconfig/esbjava2283/api.xml";
-		ESBTestCaseUtils util = new ESBTestCaseUtils();
-		relativePath = relativePath.replaceAll("[\\\\/]", "/");
-		OMElement apiConfig = util.loadResource(relativePath);
-		addApi(apiConfig);
-	}
+    public static String contentType;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+
+        super.init();
+        String relativePath = "/artifacts/ESB/synapseconfig/esbjava2283/api.xml";
+        ESBTestCaseUtils util = new ESBTestCaseUtils();
+        relativePath = relativePath.replaceAll("[\\\\/]", "/");
+        OMElement apiConfig = util.loadResource(relativePath);
```

Comment:
Please make sure this line is working properly with Window OS also. Here relativePath variable(line 54) path is set with backslash. But in loadResource function(ESBTestCaseUtils.java line number 117) concat this relativePath with system resource path which return path with forward slash for Windows OS. I think there is a conflict when you run this code in Windows machine. WDYT?